packages/create-gatsby/src/init-starter.ts

```diff
@@ -196,6 +197,59 @@ const clone = async (
   await fs.remove(path.join(rootPath, `.git`))
 
   await fs.remove(path.join(rootPath, `LICENSE`))
+
+  if (useTailwind) {
```

Comment:
This logic should also exist in `questions/styles.json`. The styling choice should have an array of files with a `source` and `targetPath`, where source is the path to the template file inside our package, targetPath the path the file should be copied to.